packages/chains/src/zetachainAthens3.ts

```diff
@@ -0,0 +1,27 @@
+import { Chain } from './types'
+
+export const zetachainAthens3 = {
```

Comment:
Should this be in sync w/ ethereum-chains? https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-7001.json

```suggestion
export const zetachainAthensTestnet = {
```

Reply:
surely, I could not locate that file myself hence missed that.